RELEASING.adoc

```diff
@@ -0,0 +1,12 @@
+= Spring Cloud GCP release process
+
+The actual release process is triggered by Pivotal.
+In order to simplify that process, here is a list of prerequisites to be performed before asking Pivotal to run the release scripts.
+
+1. Comment out the `spring-cloud-gcp-samples` module from the root `pom.xml` file, so that the samples aren't released.
+
+2. For any non-GA modules (e.g., Spanner), also comment them out from the `pom.xml` file.
```

Comment:
I don't think you need to mention any specific modules here; it would just be another thing to forget when things change in the future.